roles/check-swlib/tasks/main.yml

```diff
@@ -16,13 +16,27 @@
 - name: check-swlib | Validate base software release
   shell: |
     set -o pipefail
-    {% for i in item.files %}
-      gsutil ls -L gs://{{ swlib_mount_src }}/{{ i }} | \
-        awk '/md5/ { print $3 }'
+    {% for f in item.files %}
+    if gsutil ls gs://{{ swlib_mount_src }}/{{ f.name }} >/dev/null 2>&1; then
+      f_name="{{ f.name }}"
+      md5sum="{{ f.md5sum }}"
+    else
+      f_name="{{ f.alt_name | default('x') }}"
+      md5sum="{{ f.alt_md5sum | default('x') }}"
+    fi
+    actual_md5=$(gsutil ls -L gs://{{ swlib_mount_src }}/${f_name} | awk '/md5/ { print $3 }')
+    if [ $? -ne 0 ] ; then
+      echo "ERROR locating {{ f.name }} {{ f.alt_name | default('') }}"
+      exit 1;
```

Comment:
Can we attempt to validate all the files before exiting with failure?  As a user, it would be very frustrating to have to run multiple times when a different file now fails.

One way to do this:  remove the exit entirely here, and use failed_when to search for ERROR in stdout.